src/qml/QFieldSettings.qml

```diff
@@ -1117,7 +1117,7 @@ Page {
                   }
 
                   Label {
-                      text: qsTr( "Z values which are recorded from the positioning device will be corrected by this value. If a value of 1.6 is entered, this will result in a correction of -1.6 for each recorded value." )
+                      text: qsTr( "This value will correct the Z values recorded from the positioning device. If a value of 1.6 is entered, QField will automatically subtract 1.6 from each recorded value. Make sure to insert the effective antenna height, i.e. pole length + antenna phase centre offset." )
```

Comment:
I would defer this change to the next release, while it improves English, it breaks all translations

Reply:
@m-kuhn , removed.